.ci.yaml

```diff
@@ -264,9 +264,8 @@ targets:
 
   - name: Linux Web Engine
     recipe: engine/web_engine
-    bringup: true
+    bringup: true # https://github.com/flutter/flutter/issues/115404
```

Comment:
Can we remove bringup instead? seems like the build has been working properly.

Reply:
Done.